Add learning goals to some registry and data tutorials (#2895)
viamrobotics/docs: docs/tutorials/projects/integrating-viam-with-openai.md

no_list: true
---

<!-- LEARNING GOALS

> After following this tutorial, you will know about Viam's modular platform, know when you need to use modules, and be able to find useful resources, like the speech module and follow their instructions to use them.

Reviewer: Same nit: `resource` preferable, and slight simplification:

```suggestion
After following this tutorial, you will know about Viam's modular resources, understand when you need to use modules, and be able to find useful resources from the Viam Registry, such as the speech module.
```

> After following this tutorial, you will know about Viam's modular platform, know when you need to use modules, and be able to find useful resources, like the speech module and follow their instructions to use them.

Author: ```suggestion
After following this tutorial, you will know about modules and understand when you need to use them, and be able to find useful resources from the Viam Registry, such as the speech module.
```
